styles/colors.css

```diff
@@ -0,0 +1,4 @@
+:root {
+  --dark-blue: #2d78a3;
```

Comment:
I would name the colors like that: --color-dark-blue. Then you have the oppertunity to use autocomplition much better when you have more than two colors.